dart/lib/src/hub.dart

```diff
@@ -198,7 +194,7 @@ class Hub {
   }
 
   /// Adds a breacrumb to the current Scope
-  void addBreadcrumb(Breadcrumb crumb, {dynamic hint}) {
+  void addBreadcrumb(Breadcrumb? crumb, {dynamic hint}) {
```

Comment:
```suggestion
  void addBreadcrumb(Breadcrumb crumb, {dynamic hint}) {
```

same case as SentryEvent